src/client.ts

```diff
@@ -1184,6 +1184,11 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
       this.mutedUsers = event.me.mutes;
     }
 
+    if (event.type === 'notification.mark_read') {
+      const activeChannelKeys = Object.keys(this.activeChannels);
+      activeChannelKeys.map((activeChannelKey) => (this.activeChannels[activeChannelKey].state.unreadCount = 0));
```

Comment:
Shouldn't this be `.forEach(activeChannelKey  =>`?

Reply:
Yes, that's a better method to handle this case. Thanks, @peterdeme 😄